### ISSUE TITLE
1.7.0 RC1 dist not updated

### ISSUE THREAD

**Issue by gcarothers** [uxsolutions/bootstrap-datepicker#2118]
### Expected behaviour
Source code should match dist copy

### Actual behaviour
Dist is reasonably out of date (seems to be exactly the same as 1.6.3) and is also pointed to by main so requiring or installing 1.7.0 doesn't do anything.

### Datepicker version used

1.7.0 RC1

**Comment by gcarothers** [comment]
For example:
```
-                               if (typeof this.o.format === 'string') {
-                                       if ((String(this.element[0].value).length === String(this.o.format).length) && (String(oldDates) !== Str
ing(this.dates)))
-                                               this._trigger('changeDate');
-                        this.element.change();
-                                   } else if (String(oldDates) !== String(this.dates)) {
-                        this._trigger('changeDate');
-                        this.element.change();
-                    }
-                       }
+                               if (String(oldDates) !== String(this.dates) && fromArgs) {
+                                       this._trigger('changeDate');
+                                       this.element.change();
+                               }
+            }
                        if (!this.dates.length && oldDates.length) {
```

**Comment by acrobat** [comment]
Hi @gcarothers, can you verify that you are indeed using 1.7.0-dev (you can check you dependecy manager output or check the version variable in the datepicker `console.log($.fn.datepicker.version);`). Because the dist files were generated before the release in 6de2c8e.

**Comment by fridde** [comment]
I'd like to add that I just spent 30min debugging due to using cloudflares marked 1.7.0-RC1 that, if used, actually gives
```
> console.log($.fn.datepicker.version);
1 1.7.0-dev
```
If you check [their version of RC1](https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.7.0-RC1/js/bootstrap-datepicker.js) the content STILL resembles what @gcarothers has posted, even after 19 days of the dist-update.

This drove me crazy, since the excerpt was the exact place where @teoulas had fixed the "update-on-initialization"-bug with [pull request 1912](https://github.com/uxsolutions/bootstrap-datepicker/pull/1912).

So what I assumed was a fixed version was actually still from before the pull request from august 2016.

**Comment by acrobat** [comment]
Hmm you are right about the missing lines on the cdn version of the js file. I will look into this! I'm also trying to release RC2 this weekend so hopefully this solves the problem!

**Comment by fridde** [comment]
Not trying to annoy, but I just wonder how the release process works:

Are you just redefining [this file](https://github.com/uxsolutions/bootstrap-datepicker/blob/master/js/bootstrap-datepicker.js) as the new version and the CDNs out there detect a changed version in the dist-folder?

Or are you manually editing anything in beforehand? I was thinking about mirroring the current master branch of above mentioned file on my server and checking for updates once a day (to avoid hotlinking to github), since it's the version without the annoying "trigger change() on inititialization" bug that somehow found its way back into RC1.

**Comment by acrobat** [comment]
The release process is creating a tag in git and publishing it on npm. For now you are good to mirror master. After 1.7.0 release we branch off to 1.7 and master will be 1.8/2.0

1.7.0-RC2 should not be away long. If #2124 is fixed soon I tag RC2 otherwise this will wait for RC3 and I have RC2 out in the next few days

**Comment by vsn4ik** [comment]
Fixed by 0d32bc5d91da11d9a3587537c3c36ce7ee815c94
